custom_components/google_home/api.py

```diff
@@ -297,6 +299,57 @@ async def update_do_not_disturb(
 
         return device
 
+    async def update_alarm_volume(
+        self, device: GoogleHomeDevice, volume: float | None = None
+    ) -> GoogleHomeDevice:
+        """Gets or sets the alarm volume setting on a Google Home device."""
+
+        data: JsonDict | None = None
+        polling = False
+
+        if volume is not None:
+            # Setting is inverted on device
+            data = {JSON_ALARM_VOLUME: volume}
+            _LOGGER.debug(
+                "Setting Alarm Volume setting to %f on Google Home device %s",
+                volume,
+                device.name,
+            )
+        else:
+            polling = True
+            _LOGGER.debug(
+                "Getting Alarm Volume setting from Google Home device %s",
+                device.name,
+            )
+
+        response = await self.request(
+            method="POST",
+            endpoint=API_ENDPOINT_ALARM_VOLUME,
+            device=device,
+            data=data,
+            polling=polling,
+        )
+        if response is not None:
+            if JSON_NOTIFICATIONS_ENABLED in response:
+                volume_raw: str = str(response[JSON_ALARM_VOLUME])
+                loaded_volume: float = float(volume_raw)
```

Comment:
No need to repeat that, it'll be inferred.
```suggestion
                volume_raw = str(response[JSON_ALARM_VOLUME])
                loaded_volume = float(volume_raw)
```